quickwit/quickwit-index-management/src/index.rs

```diff
@@ -193,6 +196,46 @@ impl IndexService {
         Ok(deleted_splits)
     }
 
+    /// Deletes the indexes specified with `index_id_patterns`.
+    /// This is a wrapper of delete_index, and support index delete with index pattern
+    ///
+    /// * `index_id_patterns` - The target index Id patterns.
+    /// * `dry_run` - Should this only return a list of affected files without performing deletion.
+    pub async fn delete_indexes(
+        &self,
+        index_id_patterns: Vec<String>,
+        dry_run: bool,
+    ) -> Result<Vec<SplitInfo>, IndexServiceError> {
+        // get all index_ids by index_id_patterns
+        let indexes_metadata = self
+            .metastore()
+            .list_indexes_metadatas(ListIndexesQuery::IndexIdPatterns(index_id_patterns.clone()))
+            .await?;
+
+        if indexes_metadata.is_empty() {
+            return Err(IndexServiceError::Metastore(MetastoreError::NotFound(
+                EntityKind::Indexes {
+                    index_ids: index_id_patterns,
+                },
+            )));
+        }
+        let index_ids = indexes_metadata
+            .iter()
+            .map(|index_metadata| index_metadata.index_id())
+            .collect_vec();
+        info!(index_ids = ?index_ids);
+
+        // setup delete index tasks
+        let mut delete_index_tasks = Vec::new();
+        for index_id in index_ids {
+            delete_index_tasks.push(async move {
+                info!("delete_index:{}", index_id);
```

Comment:
I would add the info log in the delete function directly. 

The way we generally format logs are more like this:

```
info!("delete index", index_id=index_id);
```

Reply:
Good suggestion